lib/Elastica/Query/GeohashCell.php

```diff
@@ -1,10 +1,12 @@
 <?php
 namespace Elastica\Query;
 
+trigger_error('Elastica\Query\GeohashCell is deprecated.', E_USER_DEPRECATED);
+
 /**
  * Class GeohashCell.
  *
- * @link https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl-geohash-cell-query.html
+ * @link https://www.elastic.co/guide/en/elasticsearch/reference/2.4/query-dsl-geohash-cell-query.html
```

Comment:
I ran the query against 5.0 and it didn't give an error. So I assume it was only removed by accident from the documentation and is only deprecated, just as geo_distance_range.